src/contracts/interfaces/IPaymentCoordinator.sol

```diff
@@ -0,0 +1,214 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./IStrategy.sol";
+import "./IPauserRegistry.sol";
+
+/**
+ * @title Interface for the `IPaymentCoordinator` contract.
+ * @author Layr Labs, Inc.
+ * @notice Terms of Service: https://docs.eigenlayer.xyz/overview/terms-of-service
+ * @notice Allows AVSs to make "Range Payments", which get distributed amongst the AVSs' confirmed
+ * Operators and the Stakers delegated to those Operators.
+ * Calculations are performed based on the completed Range Payments, with the results posted in
+ * a Merkle root against which Stakers & Operators can make claims.
+ */
+interface IPaymentCoordinator {
+    /// STRUCTS ///
+    struct StrategyAndMultiplier {
+        IStrategy strategy;
+        // weight used to compare shares in multiple strategies against one another
+        uint96 multiplier;
+    }
+
+    struct RangePayment {
+        // Strategies & relative weights of shares in the strategies
+        StrategyAndMultiplier[] strategiesAndMultipliers;
+        IERC20 token;
+        uint256 amount;
+        uint32 startTimestamp;
+        uint32 duration;
+    }
+
+    struct DistributionRoot {
+        // merkle root of the distribution
+        bytes32 root;
+        // The timestamp until which payments have been calculated
+        uint32 paymentCalculationEndTimestamp;
+        // timestamp at which the root can be claimed against
+        uint32 activatedAt;
+    }
+
+    struct EarnerTreeMerkleLeaf {
+        address earner;
+        bytes32 earnerTokenRoot;
+    }
+
+    struct TokenTreeMerkleLeaf {
+        IERC20 token;
+        uint256 cumulativeEarnings;
+    }
+
+    struct PaymentMerkleClaim {
```

Comment:
A high-level definition of what a claim is would be helpful. Is there a distinction between quorums? ie if I'm in two quorums would i have one or two claims in the window I was a staker/operator

Reply:
No distinction of quorums. Seems we'll have some further discussion on this today